apps/studio/src/lib/db/client.ts

```diff
@@ -64,6 +65,7 @@ export interface DatabaseClient {
   getTableProperties: (table: string, schema?: string) => Promise<TableProperties | null>,
   getTableCreateScript: (table: string, schema?: string) => Promise<string>,
   getViewCreateScript: (view: string) => void,
+  getMaterializedViewCreateScript: (view: string) => void,
```

Comment:
This should return the type `Promise<String>`, and it should be optional also:

`getMaterializedViewCreateScript?:`, similar to `defaultSchema`

Reply:
The function is pretty similar to `getViewCreateScript` function so I used the same types, isn't  in that case mistake also in `getViewCreateScript` function?

Reply:
Yeah the interface is wrong for many functions! A contributor converted us to TS and some stuff is half converted :-)

Reply:
so is this conversation resolved or should I change something?